[2024-12-21T20:55:13.631+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: commiteando_cosas.commit_and_push_task scheduled__2024-12-21T20:54:00+00:00 [queued]>
[2024-12-21T20:55:13.640+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: commiteando_cosas.commit_and_push_task scheduled__2024-12-21T20:54:00+00:00 [queued]>
[2024-12-21T20:55:13.640+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-12-21T20:55:13.640+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2024-12-21T20:55:13.641+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-12-21T20:55:13.654+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): commit_and_push_task> on 2024-12-21 20:54:00+00:00
[2024-12-21T20:55:13.661+0000] {standard_task_runner.py:55} INFO - Started process 6537 to run task
[2024-12-21T20:55:13.669+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'commiteando_cosas', 'commit_and_push_task', 'scheduled__2024-12-21T20:54:00+00:00', '--job-id', '2554', '--raw', '--subdir', 'DAGS_FOLDER/elPataCommits.py', '--cfg-path', '/tmp/tmpmk12_7zc']
[2024-12-21T20:55:13.670+0000] {standard_task_runner.py:83} INFO - Job 2554: Subtask commit_and_push_task
[2024-12-21T20:55:13.735+0000] {task_command.py:376} INFO - Running <TaskInstance: commiteando_cosas.commit_and_push_task scheduled__2024-12-21T20:54:00+00:00 [running]> on host a39b43f32cb0
[2024-12-21T20:55:13.794+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=commiteando_cosas
AIRFLOW_CTX_TASK_ID=commit_and_push_task
AIRFLOW_CTX_EXECUTION_DATE=2024-12-21T20:54:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-12-21T20:54:00+00:00
[2024-12-21T20:55:13.796+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-12-21T20:55:13.797+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n        cd /root/usr &&\n        git add . &&\n        git commit -m "Commit diario: $(date \'+%Y-%m-%d\')" &&\n        git push origin main\n        ']
[2024-12-21T20:55:13.805+0000] {subprocess.py:86} INFO - Output:
[2024-12-21T20:55:13.807+0000] {subprocess.py:93} INFO - /bin/bash: line 2: cd: /root/usr: Permission denied
[2024-12-21T20:55:13.808+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-12-21T20:55:13.820+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-12-21T20:55:13.828+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=commiteando_cosas, task_id=commit_and_push_task, execution_date=20241221T205400, start_date=20241221T205513, end_date=20241221T205513
[2024-12-21T20:55:13.839+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 2554 for task commit_and_push_task (Bash command failed. The command returned a non-zero exit code 1.; 6537)
[2024-12-21T20:55:13.850+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2024-12-21T20:55:13.867+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
